### ISSUE TITLE
Meaning of "iterations" in readme 

### ISSUE THREAD

**Issue by Rhuax** [atulkum/pointer_summarizer#30]
What do you mean by "100k iterations"? Is an iteration a batch update? How is that number related to the concept of epoch?

**Comment by chan0park** [comment]
I think iteration is for the number of batch updates. You can find it from `Train.trainIters()` from `train.py`. 

To relate this iteration to the concept of epoch, you need to see how they make the batches, which you can find from `example_generator()` in `data.py`.  From the data_path, it keeps pulling the batches from data_path until they consume all files in the path. After that, they redo this generation process over the data_path again (since it is a generator) unless argument `single_pass` is set as True.

In summary, (total # of examples/batch_size) iterations would be _approximately_ the same as one epoch, but note that for the last batch it may have mixed examples.

**Comment by Rhuax** [comment]
Thanks dear! I see. I close the issue
